lib/java/com/google/android/material/textfield/TextInputLayout.java

```diff
@@ -380,6 +380,7 @@
   @Nullable private Drawable endDummyDrawable;
   private int endDummyDrawableWidth;
   private Drawable originalEditTextEndDrawable;
+  protected OnClickListener endIconOnClickListener;
```

Comment:
Why do we need this and where is it being used?

Reply:
It is used in the `EndIconDelegate#callOriginalEndIconOnClickListener` method.